keanacobarde/Bangazon
# SETUP, CLASS, AND TEST DATA CREATION

## Description
- Created necessary classes
- Created DBContext
- Implemented required dependencies in Program.cs file

## Related Issue
#2 , #3 , #4 , 

## Motivation and Context
- Required for the core functionality of the backend. Set up the database with the schema highlighted in ERD. Seeded test data to allow for baseline API testing. 

## How Can This Be Tested?
- Pull PR branch to machine.
- Implement needed dependencies as well as initializing user secrets folder with connection string to local instance of DB.
- Run commands outlined in mentioned issues.
- Check PGAdmin to ensure schema was created and test data was seeded properly. 

## Screenshots (if appropriate):
N / A 

## Types of changes
